### PR TITLE
update getPinToDuctGap to ignore holedHex

### PR DESCRIPTION
**Pull request by onufer**
## Description

The current code cannot handle it since it has no `ip`.

---

## Checklist

- [X] The code is understandable and maintainable to people beyond the author.
- [ ] Tests have been added/updated to verify that the new or changed code works.
- [X] There is no commented out code in this PR.
- [X] The commit message follows [good practices](https://terrapower.github.io/armi/developer/tooling.html).
- [X] All docstrings are still up-to-date with these changes.

### CODE REVIEW

File: armi/reactor/blocks.py
Comment (jakehader) [inline armi/reactor/blocks.py:1852]: This looks like it won't ignore the holed hexagon still. Maybe this should be `elif isinstance(...)`?

**Review by jakehader — APPROVED** [review]

**Comment by john-science** [comment]
A recent PR to the docs broke the build a little.  So I will either:

a. Wait until the morning to merge this PR, so that can be fixed first.

or 

b. Just merge this one, and as long as everything BUT the docs builds, we're fine.

**Pull request merged**
